packages/esm-patient-chart-app/src/deceased/deceased-form.scss

```diff
@@ -0,0 +1,115 @@
+@import "~@openmrs/esm-styleguide/src/vars";
+@import "~carbon-components/src/globals/scss/vars";
+@import "~carbon-components/src/globals/scss/mixins";
```

Comment:
We no longer use `import` for carbon components. According to the new release by carbon, it was recommended by carbon to use `use` instead of `import`.

You can refer this file: https://github.com/openmrs/openmrs-esm-patient-management/blob/main/packages/esm-patient-search-app/src/compact-patient-search/compact-patient-banner.scss